README.md

```diff
@@ -118,9 +118,9 @@ git commit -m "Initialize the styleguide."
 
   This is a task to deploy the static styleguide to GitHub pages.
 
-  Butler will build a Sculpin production artifact to `styleguide/output_prod` and deploy the production artifact to `gh-pages` branch of the repo defined in the `conf/butler.defaults.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]".
+  Butler will build a Sculpin production artifact to and deploy the production artifact to `gh-pages` branch of the repo defined in the `conf/butler.defaults.js`. 
 
-  You may want to create a `sculpin_site_prod.yml` to define the site URL once deployed. You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/).
+  You may want to create a `sculpin_site_prod.yml` or a `config_prod.yml` (for Spress) to define the site URL once deployed. You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/) and the configuration for Spress [here](http://spress.yosymfony.com/docs/configuration/).
```

Comment:
@labbydev I know that the language is probably really clear to a developer, but to me, it was not evident how or why I should create this yml file. Heck, I skimmed this, didn't understand it, and forgot about it. Although the links to documentation are helpful, I'd prefer a simpler and more concise explanation with clear instructions, like: 

You will need to create a a `config_prod.yml` file to define the GitHub Pages site URL so that you can deploy to GitHub Pages. The `config_prod.yml` should go in the styleguide folder. Copy and paste this code into the `config_prod.yml` file:

```
# Site configuration

url: 'https://palantirnet.github.io/[your project here]'
```

You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/) and the configuration for Spress [here](http://spress.yosymfony.com/docs/configuration/).